runtime/worker.h

```diff
@@ -39,12 +37,14 @@ struct __cilkrts_worker {
     // T, H, and E pointers in the THE protocol.
     // T and E are frequently accessed and should be in a hot cache line.
     // H could be moved elsewhere because it is only touched when stealing.
-    _Atomic(struct __cilkrts_stack_frame **) tail;
-    _Atomic(struct __cilkrts_stack_frame **) exc __attribute__((aligned(64)));
-    _Atomic(struct __cilkrts_stack_frame **) head __attribute__((aligned(CILK_CACHE_LINE)));
+    std::atomic<struct __cilkrts_stack_frame **> tail;
+    std::atomic<struct __cilkrts_stack_frame **> exc
+      __attribute__((aligned(64)));
+    std::atomic<struct __cilkrts_stack_frame **> head
+      __attribute__((aligned(64)));
```

Comment:
Any particular reason for the change in alignment?  I thought there was a performance advantage to using `CILK_CACHE_LINE` alignment on `head`.